The tests are failing:

```
 =========================== short test summary info ============================
FAILED tests/components/smartthings/test_climate.py::test_all_entities[virtual_thermostat] - AssertionError
FAILED tests/components/smartthings/test_climate.py::test_all_entities[ecobee_thermostat] - AssertionError
FAILED tests/components/smartthings/test_climate.py::test_all_entities[heatit_ztrm3_thermostat] - AssertionError
FAILED tests/components/smartthings/test_climate.py::test_all_entities[bosch_radiator_thermostat_ii] - AssertionError
FAILED tests/components/smartthings/test_climate.py::test_thermostat_state_attributes_update[hvac_modes-virtual_thermostat] - AssertionError: assert [<HVACMode.AUTO: 'auto'>] == []
```

--- a/tests/components/smartthings/snapshots/test_climate.ambr
+++ b/tests/components/smartthings/snapshots/test_climate.ambr
@@ -70,6 +70,7 @@
     'area_id': None,
     'capabilities': dict({
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.HEAT: 'heat'>,
       ]),
       'max_temp': 35,
@@ -109,6 +110,7 @@
       'current_temperature': 23.9,
       'friendly_name': 'Radiator Thermostat II [+M] Wohnzimmer',
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.HEAT: 'heat'>,
       ]),
       'max_temp': 35,
@@ -431,6 +433,7 @@
         'auto',
       ]),
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.COOL: 'cool'>,
         <HVACMode.HEAT_COOL: 'heat_cool'>,
       ]),
@@ -478,6 +481,7 @@
       'friendly_name': 'Main Floor',
       'hvac_action': <HVACAction.HEATING: 'heating'>,
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.COOL: 'cool'>,
         <HVACMode.HEAT_COOL: 'heat_cool'>,
       ]),
@@ -628,6 +632,7 @@
     'area_id': None,
     'capabilities': dict({
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.HEAT: 'heat'>,
       ]),
       'max_temp': 35,
@@ -668,6 +673,7 @@
       'friendly_name': 'Hall thermostat',
       'hvac_action': <HVACAction.HEATING: 'heating'>,
       'hvac_modes': list([
+        <HVACMode.OFF: 'off'>,
         <HVACMode.HEAT: 'heat'>,
       ]),
       'max_temp': 35,
@@ -695,6 +701,7 @@
         'on',
       ]),
       'hvac_modes': list([
+        <HVACMode.AUTO: 'auto'>,
       ]),
       'max_temp': 35.0,
       'min_temp': 7.0,
@@ -738,6 +745,7 @@
       'friendly_name': 'asd',
       'hvac_action': <HVACAction.COOLING: 'cooling'>,
       'hvac_modes': list([
+        <HVACMode.AUTO: 'auto'>,
       ]),
       'max_temp': 35.0,
       'min_temp': 7.0,

--- a/tests/components/smartthings/test_climate.py
+++ b/tests/components/smartthings/test_climate.py
@@ -817,10 +817,10 @@ async def test_updating_humidity(
         (
             Capability.THERMOSTAT_MODE,
             Attribute.SUPPORTED_THERMOSTAT_MODES,
-            ["coolClean", "dryClean"],
+            ["rush hour", "heat"],
             ATTR_HVAC_MODES,
             [HVACMode.AUTO],
-            [],
+            [HVACMode.AUTO, HVACMode.HEAT],
         ),
     ],
     ids=[

--- a/tests/components/smartthings/test_climate.py
+++ b/tests/components/smartthings/test_climate.py
@@ -819,8 +819,8 @@ async def test_updating_humidity(
             Attribute.SUPPORTED_THERMOSTAT_MODES,
             ["coolClean", "dryClean"],
             ATTR_HVAC_MODES,
+            [HVACMode.AUTO],
             [],
-            [HVACMode.COOL, HVACMode.DRY],
         ),
     ],
     ids=[

